proto/cosmos/upgrade/v1beta1/query.proto

```diff
@@ -69,9 +69,11 @@ message QueryUpgradedConsensusStateRequest {
 // QueryUpgradedConsensusStateResponse is the response type for the Query/UpgradedConsensusState
 // RPC method.
 message QueryUpgradedConsensusStateResponse {
-  reserved 1;
+  // Deprecated: Please use the bytes field instead. This field will always be
+  // empty in v0.43+.
+  google.protobuf.Any upgraded_consensus_state = 1 [deprecated = true];
 
-  bytes upgraded_consensus_state = 2;
+  bytes upgraded_consensus_state_bytes = 2;
```

Comment:
We cannot have two fields with same name. I propose to:
- keep `upgraded_consensus_state` for the Any field, same as in 0.42
- rename to new bytes field to `upgraded_consensus_state_bytes`.

This would be a breaking change in the RCs, and ibc-go might need to change some code. cc @colin-axner

Reply:
This is a breaking change in 0.43.0-RC -> 0.43.0-RC3.

I know it sounds "hacky" - but this thing was only added for IBC and only IBC uses it. So maybe we can keep the original breaking change (from https://github.com/cosmos/cosmos-sdk/pull/8673/files#r682168269)?

Reply:
I'm fairly certain ibc-go doesn't rely on this gRPC and relayers don't either. Relayers use ABCI queries since they need proof of the consensus state. This is just a convenience function for the cli